### DISCUSSION TITLE
No confirmation for file/folder deletion

### DISCUSSION THREAD

**Discussion by fatualux** [kamiyaa/joshuto#255] [Q&A]
Is it possible to set a "no confirmation option" for item/multiple items deletion?
I searched for this option in the documentation pages but found nothing about it.
Thank you for your support

**Comment by superiums** [comment]
maybe it's a good idea to map it to a complex key binding, 
such as `dD`

**Reply by fatualux** [reply]
At the moment when I delete an item with dD (I always do so, never press Del), it asks for a confirmation (I have to press y); to delete multiple items I have to press yy

**Reply by superiums** [reply]
that's OK.
